docs/index.rst

```diff
@@ -35,8 +35,8 @@ Getting help
 
 If you're having trouble with something, these resources might help.
 
-- Try the :doc:`faq` first, it's got answers to all common questions.
-- Ask us and hang out with us in our :resource:`Discord <discord>` server.
+- Try the :doc:`FAQs` first, it's got answers to all common questions.
```

Comment:
does this doc link actually resolve, you can probably name it `` :doc:`FAQs <faq>` ``